.bazelrc

```diff
@@ -51,7 +50,7 @@ test --incompatible_strict_action_env
 build --experimental_remote_merkle_tree_cache
 
 # Ensure that tags applied in BUILDs propagate to actions
-build --incompatible_allow_tags_propagation
+build --experimental_allow_tags_propagation
```

Comment:
This was renamed quite some time ago. `incompatible_allow_tags_propagation` still works but will be removed eventually

Reply:
I wonder if we still need it— we don't use it in other repos AFAIK. Nothing to worry here though, just bring it up.

Reply:
It's a new/experimental feature that will hopefully eventually become the default. I'm not sure if it's required for anything in this repo but I've started using it.

Reply:
Okay, not opposed to keeping it here, just wanted to see if if we actively rely on it